### ISSUE TITLE
style_encoder optimizer step

### ISSUE THREAD

**Issue by thephucnguyen** [clovaai/stargan-v2#80]
Hi, 
I am a big fan of you. I follow style transfer from StarGAN and think you did a great work.
After digging in the code, there is a small part I am quite confused.
In core/solver.py 
`# train the generator
            g_loss, g_losses_latent = compute_g_loss(
                nets, args, x_real, y_org, y_trg, z_trgs=[z_trg, z_trg2], masks=masks)
            self._reset_grad()
            g_loss.backward()
            optims.generator.step()
            optims.mapping_network.step()
            optims.style_encoder.step()

            g_loss, g_losses_ref = compute_g_loss(
                nets, args, x_real, y_org, y_trg, x_refs=[x_ref, x_ref2], masks=masks)
            self._reset_grad()
            g_loss.backward()
            optims.generator.step()`
I see in the code you put optims.style_encoder.step() righ after you calculate loss from z_trgs.
So I wonder why don't we put optims.style_encoder.step() after we calculate loss from x_refs?
And are they different?
Thank you
